{
	"document": {
		"aggregate_severity": {
			"namespace": "https://www.suse.com/support/security/rating/",
			"text": "moderate"
		},
		"category": "csaf_security_advisory",
		"csaf_version": "2.0",
		"distribution": {
			"text": "Copyright 2024 SUSE LLC. All rights reserved.",
			"tlp": {
				"label": "WHITE",
				"url": "https://www.first.org/tlp/"
			}
		},
		"lang": "en",
		"notes": [
			{
				"category": "summary",
				"text": "Security update for python-Django1",
				"title": "Title of the patch"
			},
			{
				"category": "description",
				"text": "This update for python-Django1 fixes the following issues:\n\n- CVE-2019-12308: Fixed an improper validatation of the clickable 'Current URL' link in AdminURLFieldWidget \n  which could have allowed attackers to perform XSS attacks (bsc#1136468).\n",
				"title": "Description of the patch"
			},
			{
				"category": "details",
				"text": "SUSE-2019-2034,SUSE-OpenStack-Cloud-9-2019-2034,SUSE-OpenStack-Cloud-Crowbar-9-2019-2034",
				"title": "Patchnames"
			},
			{
				"category": "legal_disclaimer",
				"text": "CSAF 2.0 data is provided by SUSE under the Creative Commons License 4.0 with Attribution (CC-BY-4.0).",
				"title": "Terms of use"
			}
		],
		"publisher": {
			"category": "vendor",
			"contact_details": "https://www.suse.com/support/security/contact/",
			"name": "SUSE Product Security Team",
			"namespace": "https://www.suse.com/"
		},
		"references": [
			{
				"category": "external",
				"summary": "SUSE ratings",
				"url": "https://www.suse.com/support/security/rating/"
			},
			{
				"category": "self",
				"summary": "URL of this CSAF notice",
				"url": "https://ftp.suse.com/pub/projects/security/csaf/suse-su-2019_2034-1.json"
			},
			{
				"category": "self",
				"summary": "URL for SUSE-SU-2019:2034-1",
				"url": "https://www.suse.com/support/update/announcement/2019/suse-su-20192034-1/"
			},
			{
				"category": "self",
				"summary": "E-Mail link for SUSE-SU-2019:2034-1",
				"url": "https://lists.suse.com/pipermail/sle-security-updates/2019-August/005769.html"
			},
			{
				"category": "self",
				"summary": "SUSE Bug 1136468",
				"url": "https://bugzilla.suse.com/1136468"
			},
			{
				"category": "self",
				"summary": "SUSE CVE CVE-2019-12308 page",
				"url": "https://www.suse.com/security/cve/CVE-2019-12308/"
			}
		],
		"title": "Security update for python-Django1",
		"tracking": {
			"current_release_date": "2019-08-01T14:25:23Z",
			"generator": {
				"date": "2019-08-01T14:25:23Z",
				"engine": {
					"name": "cve-database.git:bin/generate-csaf.pl",
					"version": "1"
				}
			},
			"id": "SUSE-SU-2019:2034-1",
			"initial_release_date": "2019-08-01T14:25:23Z",
			"revision_history": [
				{
					"date": "2019-08-01T14:25:23Z",
					"number": "1",
					"summary": "Current version"
				}
			],
			"status": "final",
			"version": "1"
		}
	},
	"product_tree": {
		"branches": [
			{
				"branches": [
					{
						"branches": [
							{
								"category": "product_version",
								"name": "python-Django1-1.11.20-3.6.1.noarch",
								"product": {
									"name": "python-Django1-1.11.20-3.6.1.noarch",
									"product_id": "python-Django1-1.11.20-3.6.1.noarch"
								}
							},
							{
								"category": "product_version",
								"name": "python3-Django1-1.11.20-3.6.1.noarch",
								"product": {
									"name": "python3-Django1-1.11.20-3.6.1.noarch",
									"product_id": "python3-Django1-1.11.20-3.6.1.noarch"
								}
							}
						],
						"category": "architecture",
						"name": "noarch"
					},
					{
						"branches": [
							{
								"category": "product_name",
								"name": "SUSE OpenStack Cloud 9",
								"product": {
									"name": "SUSE OpenStack Cloud 9",
									"product_id": "SUSE OpenStack Cloud 9",
									"product_identification_helper": {
										"cpe": "cpe:/o:suse:suse-openstack-cloud:9"
									}
								}
							},
							{
								"category": "product_name",
								"name": "SUSE OpenStack Cloud Crowbar 9",
								"product": {
									"name": "SUSE OpenStack Cloud Crowbar 9",
									"product_id": "SUSE OpenStack Cloud Crowbar 9",
									"product_identification_helper": {
										"cpe": "cpe:/o:suse:suse-openstack-cloud-crowbar:9"
									}
								}
							}
						],
						"category": "product_family",
						"name": "SUSE Linux Enterprise"
					}
				],
				"category": "vendor",
				"name": "SUSE"
			}
		],
		"relationships": [
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python-Django1-1.11.20-3.6.1.noarch as component of SUSE OpenStack Cloud 9",
					"product_id": "SUSE OpenStack Cloud 9:python-Django1-1.11.20-3.6.1.noarch"
				},
				"product_reference": "python-Django1-1.11.20-3.6.1.noarch",
				"relates_to_product_reference": "SUSE OpenStack Cloud 9"
			},
			{
				"category": "default_component_of",
				"full_product_name": {
					"name": "python-Django1-1.11.20-3.6.1.noarch as component of SUSE OpenStack Cloud Crowbar 9",
					"product_id": "SUSE OpenStack Cloud Crowbar 9:python-Django1-1.11.20-3.6.1.noarch"
				},
				"product_reference": "python-Django1-1.11.20-3.6.1.noarch",
				"relates_to_product_reference": "SUSE OpenStack Cloud Crowbar 9"
			}
		]
	},
	"vulnerabilities": [
		{
			"cve": "CVE-2019-12308",
			"ids": [
				{
					"system_name": "SUSE CVE Page",
					"text": "https://www.suse.com/security/cve/CVE-2019-12308"
				}
			],
			"notes": [
				{
					"category": "general",
					"text": "An issue was discovered in Django 1.11 before 1.11.21, 2.1 before 2.1.9, and 2.2 before 2.2.2. The clickable Current URL value displayed by the AdminURLFieldWidget displays the provided value without validating it as a safe URL. Thus, an unvalidated value stored in the database, or a value provided as a URL query parameter payload, could result in an clickable JavaScript link.",
					"title": "CVE description"
				}
			],
			"product_status": {
				"fixed": [
					"SUSE OpenStack Cloud 9:python-Django1-1.11.20-3.6.1.noarch",
					"SUSE OpenStack Cloud Crowbar 9:python-Django1-1.11.20-3.6.1.noarch"
				]
			},
			"references": [
				{
					"category": "external",
					"summary": "CVE-2019-12308",
					"url": "https://www.suse.com/security/cve/CVE-2019-12308"
				},
				{
					"category": "external",
					"summary": "SUSE Bug 1136468 for CVE-2019-12308",
					"url": "https://bugzilla.suse.com/1136468"
				}
			],
			"remediations": [
				{
					"category": "vendor_fix",
					"details": "To install this SUSE Security Update use the SUSE recommended installation methods like YaST online_update or \"zypper patch\".\n",
					"product_ids": [
						"SUSE OpenStack Cloud 9:python-Django1-1.11.20-3.6.1.noarch",
						"SUSE OpenStack Cloud Crowbar 9:python-Django1-1.11.20-3.6.1.noarch"
					]
				}
			],
			"scores": [
				{
					"cvss_v3": {
						"baseScore": 6.1,
						"baseSeverity": "MEDIUM",
						"vectorString": "CVSS:3.0/AV:N/AC:L/PR:N/UI:R/S:C/C:L/I:L/A:N",
						"version": "3.0"
					},
					"products": [
						"SUSE OpenStack Cloud 9:python-Django1-1.11.20-3.6.1.noarch",
						"SUSE OpenStack Cloud Crowbar 9:python-Django1-1.11.20-3.6.1.noarch"
					]
				}
			],
			"threats": [
				{
					"category": "impact",
					"date": "2019-08-01T14:25:23Z",
					"details": "moderate"
				}
			],
			"title": "CVE-2019-12308"
		}
	]
}
